tools/lib/h5tools_dump.c

```diff
@@ -2241,9 +2238,9 @@ h5tools_print_datatype(FILE *stream, h5tools_str_t *buffer, const h5tool_format_
                 else
                     order_s = "";
 
-                /* print size and byte order */
-                h5tools_str_append(buffer, "%lu-bit%s floating-point", (unsigned long)(8 * H5Tget_size(type)),
-                                   order_s);
+                /* print size. byte order, and precision */
```

Comment:
This is helpful, and not sufficient for floating-point types.  There's a lot more details for floating point types: mantissa, sign bit, etc.

Reply:
Yes, but how much detail is needed? h5ls does print more and I could make h5dump match h5ls - at the very least.

Reply:
What's the output from h5ls look like?

Reply:
Opened "tldouble.h5" with sec2 driver.
dset                     Dataset {3/3}
    Location:  1:800
    Links:     1
    Storage:   48 logical bytes, 48 allocated bytes, 100.00% utilization
    Type:      128-bit little-endian floating-point
               (80 bits of precision beginning at bit 0)
               (48 zero bits at bit 80)
               (significant for 64 bits at bit 0, no normalization)
               (exponent for 15 bits at bit 64, bias is 0x3fff)
               (sign bit at 79)